recipes/libxsmm/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "libxsmm" %}
+{% set version = "1.13" %}
```

Comment:
There is no information about API/ABI stability at https://abi-laboratory.pro/?view=timeline&l=libxsmm

Do you know if this lib is stable in major or minor version only?
I think it would also be good to include run_exports ...

Reply:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements

Have a look at https://github.com/conda-forge/zlib-feedstock/blob/master/recipe/meta.yaml#L13 for an example.